File: basicProgramming/c++/ifEsle/p2.cpp
#include <iostream>
using namespace std;

int main()
{
    int num;
    cout << "Enter any number" << endl;
    cin >> num;
    if (num % 2 == 0)
    {
        cout << "Even";
    }
    else
    {
        cout << "Odd";
    }
    return 0;
}

File: basicProgramming/dart/ifElse/p2.dart
//!Write a program to check if a number is even or odd.

import 'dart:io';

void main() {
  print('Enter any number');
  int num = int.parse(stdin.readLineSync()!);

  if (num % 2 == 0) {
    print("Even");
  } else {
    print("Odd");
  }
}
